### ISSUE TITLE
Update code descriptions in referenceRelationType codelist

### ISSUE THREAD

**Issue by ghost** [Open-Telecoms-Data/open-fibre-data-standard#128]
I was expecting it to be the same name. 

Also generally links to "[referenceRelationType codelist](https://open-fibre-data-standard.readthedocs.io/en/latest/reference/codelists.html#referenceRelationType)." are broken - they go to the correct page but not the correct heading.

(For discussion: that code list being open not closed might cause problems for automatic tools later, what if someone decides to add a "tag:opentelecomdata.net,2022:organisationsFile" option but the tool doesn't know to get it and look for orgs there?)

**Comment by duncandewhurst** [comment]
Yes, that is the intended name of both the field and the codelist. What were you expecting it to be the same as?

https://github.com/Open-Telecoms-Data/open-fibre-data-standard/pull/129 fixes the broken link. Thanks for flagging!

I'll leave this issue open to discuss whether the codelist should be open or closed.

**Comment by lgs85** [comment]
I've updated the title to reflect the issue we need to consider for Beta

**Comment by ghost** [comment]
I was expecting "relatedResources/0/rel" to point to a codelist called "Related Resource codelist" - "Reference relation codelist" is just different enough that it caused me to stop and think "eh, is that difference important and meant to be telling me something?" Maybe if the link hadn't also been broken it wouldn't have caused me problems.

**Comment by duncandewhurst** [comment]
Ah, I see. I think that this will become clearer once `relatedResources` is renamed to links in https://github.com/Open-Telecoms-Data/open-fibre-data-standard/issues/83. We can also reword the code descriptions so they read more like relation types than resource types. Most codes in https://www.iana.org/assignments/link-relations/link-relations.xhtml#link-relations-1 start with "Refers to..."

**Comment by duncandewhurst** [comment]
I've split out the open/closed question into a separate issue.
